engine/src/main/java/org/hibernate/validator/internal/constraintvalidators/bv/DecimalMinValidatorForCharSequence.java

```diff
@@ -15,6 +15,9 @@
 import org.hibernate.validator.internal.util.logging.LoggerFactory;
 
 /**
+ * Check that the character sequence (e.g. string) being validated represents a number, and has a value
```

Comment:
This class was missing a javadoc altogether, so I created one based on the `DecimalMaxValidatorForCharSequence`.